src/common.c

```diff
@@ -151,24 +151,34 @@ size_t hexlenl(const char *q, int *extra_chars)
 	return _hexlen(q, atoi16l, extra_chars);
 }
 
+static int isdec_len(const char *q, const char *mxv)
+{
+	const char *p = q;
+	do
+	{
```

Comment:
We don't normally put braces on their own lines, except for braces around function bodies. So this would be:

    do {